clr-k8s-examples/DEVELOP.md

```diff
@@ -1,6 +1,33 @@
 # Developer
 
-This document describes the key concepts and technologies used in the project.
+This document describes the key concepts and technologies used in the project and how to contribute.
+
+## Code Conventions
+
+### Shell Scripts 
+
+Shell scripts should adhere to the [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) as much as possible.
+
+#### Formatting with `shfmt`
+
+The [shfmt](https://github.com/mvdan/sh#shfmt) tool should be used to format shell scripts with 2 spaces and uses the following parameters:
```

Comment:
The [shfmt](https://github.com/mvdan/sh#shfmt) tool should be used to format shell scripts with 2 spaces and use the following parameters:

Reply:
went with 'should use'. thanks!